data/style.scss

```diff
@@ -131,7 +156,7 @@ nav {
     justify-content: flex-end;
 }
 
-nav > div {
+nav>div {
```

Comment:
All this formatting was done automatically by VSCode, all of which I believe is correct from a best-practice POV. Let me know if you are okay with keeping those changes here.

Reply:
I suppose it's fine but how about you make another PR which I'll merge before this one where you _only_ reformat the scss? It's going to be cleaner for the history then and will minimize the differences here.